src/services/session-store.js

```diff
@@ -52,8 +52,20 @@ const clearOtherSessions = async (userId, currentSessionId) => {
     console.log('[Session] Clear Result:', results);
 };
 
+const clearAllSessions = async (userId, currentSessionId) => {
```

Comment:
```suggestion
const clearAllSessions = async (userId) => {
```
No need for `currentSessionId` if we need to clear all sessions for the user.